src/css/_variables.scss

```diff
@@ -1,3 +1,6 @@
 $baseline: 12px;
 $gutter: 10px;
 $shadow-height: 18px;
+
+$brexit-control-button-size: 28px;
```

Comment:
if it makes sense, I'd prefer these went near the things they're relevant too (assuming that those things are in one file)